manage-images.md

```diff
@@ -0,0 +1,120 @@
+An image is the base for a container running in the Anbox Cloud. It contains all necessary components, like Anbox or the Android root file system. Each release of Anbox Cloud comes with an update image, which must be uploaded to the AMS service to make it available to be used inside any container.
+
+AMS can manage different types of images, which can be individually selected by applications. For that, each image must be named when it's initially uploaded to AMS. If the same image receives an update with the next release of Anbox Cloud, it does not need to be re-created but can simply be updated. This will cause all applications using the image to be updated and rebased on the new image version.
```

Comment:
As mentioned above, images already have a name when they are downloaded by AMS, the user doesn't have to do anything.

Unrelated, but would it be worth mentioning that the *types* of images depend on the android version as well as the system architecture or would that be too much details ?

Reply:
We should have some more information on instance types, what they mean and how you choose the suitable one - but maybe that should go into the Instance types topic.
Or is there an "image type" in addition to an "instance type"?

Reply:
Good point, we have a reference for instance types but it's a bit lacking

In depends on what you mean by "image type". The different images we have are based on different android versions and different architectures (amd64, arm64) so you could say those are different types of images. However they are completely separate from instance-types

Reply:
OK, now I understand your original comment. Let's get rid of the term "types" here, it's confusing.